libgdf/include/gdf/cffi/types.h

```diff
@@ -6,6 +6,7 @@ typedef unsigned char gdf_valid_type;
 typedef	long	gdf_date64;
 typedef	int		gdf_date32;
 typedef	int		gdf_category;
+typedef	long	gdf_timestamp;
```

Comment:
Indentation issue here or on previous two lines.

Reply:
There are indentation issues all over  the place (perhaps taken care off by the large refactor?); you'll notice that even with the three earlier lines it exists. IMHO it's better not to fix indentation issues in code that is not touched, and save that for a different PR.